src/shared/verbs.c

```diff
@@ -45,10 +45,23 @@ bool running_in_chroot_or_offline(void) {
         return r > 0;
 }
 
+const Verb* verbs_find_verb(const char *name, const Verb verbs[]) {
+        for (unsigned i = 0;; i++) {
```

Comment:
we iterate through memory here, please use `size_t`

Reply:
I was just moving existing code... Will fix.